src/utils/getAgentName.js

```diff
@@ -16,16 +16,28 @@ const AGENTS = [
   'sdk',
 ];
 
+// keeping the naming streamlined and in matching with previous data
+const RENAMES = { '.net': 'dotnet', node: 'nodejs' };
+
 /**
  * @param {string} subject The release note "subject" in frontmatter
  * @returns {string}
  */
-const getAgentName = (subject) =>
-  subject &&
-  AGENTS.find(
-    (agent) =>
-      subject.toLowerCase().includes(agent) &&
-      !subject.toLowerCase().includes('insights')
-  );
+
+const renameKeys = Object.keys(RENAMES);
+
+const getAgentName = (subject) => {
+  if (subject) {
+    const subjLowercase = subject.toLowerCase();
+    const agentName = AGENTS.find(
+      (agent) =>
+        subjLowercase.includes(agent) && !subjLowercase.includes('insights')
+    );
+
+    return agentName && renameKeys.includes(agentName)
+      ? RENAMES[agentName]
+      : agentName;
```

Comment:
You might be able to simplify this with the [in operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in):

```js
return agentName in RENAMES ? RENAME[agentName] : agentName;
```

Or perhaps even shorter by using the [short-circuit OR evaluation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#short-circuit_evaluation):

```js
return RENAMES[agentName] || agentName;
```

I'm not 100% that the second example works, might require a little tweaking.

Reply:
ooh yeah! that `in` is nice, and i think the or should totally work